arangod/IResearch/IResearchFeature.cpp

```diff
@@ -787,6 +792,7 @@ std::string const CONSOLIDATION_THREADS_IDLE_PARAM(
 std::string const FAIL_ON_OUT_OF_SYNC(
     "--arangosearch.fail-queries-on-out-of-sync");
 std::string const SKIP_RECOVERY("--arangosearch.skip-recovery");
+std::string const CACHE_LIMIT("--arangosearch.columns_cache_limit");
```

Comment:
Our option names don't use underscores, but hyphens.
```suggestion
std::string const CACHE_LIMIT("--arangosearch.columns-cache-limit");
```